query/ad_third_party_application_not_allowed.sql

```diff
@@ -0,0 +1,17 @@
+-- Ensure third party integrated applications are not allowed
+select
+  -- Required columns
+  tenant_id as resource,
+  case
+    when not (default_user_role_permissions -> 'allowedToCreateApps')::bool then 'ok'
+    else 'alarm'
+  end as status,
+  case
+     when not (default_user_role_permissions -> 'allowedToCreateApps')::bool then 'third party integrated applications are not allowed.'
```

Comment:
```suggestion
     when not (default_user_role_permissions -> 'allowedToCreateApps')::bool then 'third party integrated applications not allowed.'
```
We may avoid using linking verbs